src/ariel-os-macros/src/config.rs

```diff
@@ -39,8 +39,8 @@
 ///
 /// # Panics
 ///
-/// This macro panics when the `ariel-os` crate cannot be found as a dependency of the crate where
-/// this macro is used.
+/// This macro panics when the `ariel-os` crate (or `ariel-os-embassy`, for use in crates which
+/// `ariel-os` depends on) cannot be found as a dependency of the crate where this macro is used.
```

Comment:
(Needs to be reflowed properly:)
```suggestion
/// The `ariel-os` crate (or `ariel-os-embassy`, for use in crates which
/// `ariel-os` depends on) must be a dependency of the crate where this macro is used, otherwise this macro panics.
```